src/test_l1_dsHdmiIn.c

```diff
@@ -1598,17 +1641,18 @@ void test_l1_dsHdmiIn_positive_dsIsHdmiARCPort_sink(void) {
  * 
  */
 void test_l1_dsHdmiIn_negative_dsIsHdmiARCPort_sink(void) {
+
     gTestID = 32;
     UT_LOG("\n In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID);
+    bool isArcPort;
```

Comment:
isArcPort required to get initialize with default value

Reply:
Done